packages/extension-chakra-storefront/src/components/forms/profile-fields.jsx

```diff
@@ -20,8 +20,8 @@ const ProfileFields = ({form, prefix = ''}) => {
     })
 
     return (
-        <Stack spacing={5} aria-label={intl.formatMessage(formTitleAriaLabel)}>
-            <SimpleGrid columns={[1, 1, 1, 2]} spacing={5}>
+        <Stack gap={5} aria-label={intl.formatMessage(formTitleAriaLabel)}>
+            <SimpleGrid columns={[1, 1, 1, 2]} gap={5}>
```

Comment:
Non-blocking comment: Should we use string as prop for these? I think last time you mention using string has more benefit when using js expression. 
```suggestion
            <SimpleGrid columns={['1', '1', '1', '2']} gap='5'>
```

Reply:
I will change gap to use string literal. But I don't think we should changes columns as it inherently represents a number not string (number of columns)